src/reactions/livestream-reaction.ts

```diff
@@ -0,0 +1,34 @@
+import { logger } from '../utilities/logger';
+import { ReactionDef } from './reaction-def';
+
+export const liveStreamReaction: ReactionDef = {
+  emoji: '🎥',
+  description: 'Adds the livestream role to the user who reacts',
+  command: async (reaction, { config, user }) => {
+    const streamRole = reaction.message.guild?.roles.cache.find(
+      (role) => role.name === config.STREAM_NOTIF_ROLE
+    );
+    if (!streamRole) {
+      logger.warn('Stream role not found');
+      return;
+    }
+    if (reaction.message.id !== config.STREAM_MSG_ID) {
+      return;
+    }
+    if (user.partial) {
+      user = await user.fetch();
+    }
+    const target = reaction.message.guild?.member(user);
+    if (!target) {
+      logger.warn('user error');
+      return;
+    }
+    if (target.roles.cache.find((r) => r === streamRole)) {
+      target.roles.remove(streamRole).catch((e) => logger.error(e));
+      reaction.message.reactions.cache.get('🎥')?.remove();
```

Comment:
Is it part of the requirement to remove this instantly? Or should we actually keep it and maybe **remove** the user if they "un-react" to the message?

Reply:
Do we want to do this refactor?

Reply:
as discussed we can add this later